packages/mc-scripts/config/create-webpack-config-for-development.js

```diff
@@ -227,8 +227,10 @@ module.exports = ({
               plugins: () => [
                 postcssImport({ path: sourceFolders }),
                 postcssPresetEnv({
-                  browsers: browserslist.development,
-                  autoprefixer: { grid: true },
+                  autoprefixer: {
+                    grid: true,
+                    overrideBrowserslist: browserslist.development,
```

Comment:
Nice. What does it use by default?

Reply:
<img width="875" alt="image" src="https://user-images.githubusercontent.com/1110551/59368753-339e9e00-8d3f-11e9-9ca1-9855540a0f65.png">

(see last line)

> If the above methods did not produce a valid result Browserslist will use defaults: `> 0.5%, last 2 versions, Firefox ESR, not dead`

Reply:
Confusing though as your package.json has the mentioned entry. Refactored from a custom config.